apps/prairielearn/elements/pl-file-editor/pl-file-editor.py

```diff
@@ -168,7 +168,8 @@ def parse(element_html: str, data: pl.QuestionData) -> None:
     # We will store the files in the submitted_answer["_files"] key,
     # so delete the original submitted answer format to avoid
     # duplication
-    del data["submitted_answers"][answer_name]
+    if answer_name in data["submitted_answers"]:
+        del data["submitted_answers"][answer_name]
```

Comment:
Nit, one-liner maybe?

```suggestion
    data["submitted_answers"].pop(answer_name, None)
```